examples/gallery/grid/track_sampling.py

```diff
@@ -0,0 +1,38 @@
+"""
+Sampling along tracks
+---------------------
+
+The :meth:`pygmt.grdtrack` method samples a raster grid's value along specified points.
+We will need to input a 2D raster to ``grid`` which can be an ``xarray.DataArray``. The
+``points`` parameter can be a ``pandas.DataFrame`` table where the first two columns are
+x and y (or longitude and latitude). Note also that there is a ``newcolname`` parameter
+to set that will be used to name the new column of values we sampled from the grid.
+
+Alternatively, we can provide a NetCDF file path to ``grid``. An ASCII file path can
+also be accepted for ``points``, but an ``outfile`` parameter will then need to be set
+to name the resulting output ASCII file.
+"""
+
+import pygmt
+
+# Load sample grid and point datasets
+grid = pygmt.datasets.load_earth_relief()
+points = pygmt.datasets.load_ocean_ridge_points()
+# Sample the bathymetry along the world's ocean ridges at specified track points
+track = pygmt.grdtrack(points=points, grid=grid, newcolname="bathymetry")
+
+fig = pygmt.Figure()
+# Plot the earth relief grid on Cylindrical Stereographic projection, masking land areas
+fig.basemap(region="g", frame=True, projection="Cyl_stere/30/-20/8i")
+fig.grdimage(grid=grid, cmap="ocean")
```

Comment:
It might be best to plot this with a gray colormap and the points in color to make them stand out a bit more.

Reply:
Okay, I've made the background gray and kept the same colormap for the points but made them a little bigger so it's easier to see. Also thought it might be nice to have a Pacific-centered map :wink:

![image](https://user-images.githubusercontent.com/23487320/59158150-6aae4e80-8ab6-11e9-9a61-7f4805461f73.png)